common/genes/05_genes_special_accessories_clothes.txt

```diff
@@ -1361,6 +1361,18 @@ special_genes = {
 				boy = male
```

Comment:
See these black lines? Looks like displaced normal map to me.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120908156-9c902680-c678-11eb-9ab5-f7a300479f30.png)

</details>